app/forms/validation/mappings/Formatters.scala

```diff
@@ -110,4 +110,43 @@ trait Formatters {
         baseFormatter.unbind(key, value.getOrElse("").toString)
     }
 
+  private[mappings] def optionCurrencyFormatterWithMinCheck(requiredKey: String,
```

Comment:
Could you share some code with `optionCurrencyFormatter`?

Reply:
Yes I have fixed it, now both conditions use optionCurrencyFormatter

Reply:
Good. Confirmed.